<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IN1-1019286LT" name="Sign in Testcases" tests="2" failures="1" timestamp="30 Mar 2021 04:22:11 GMT" time="72.924" errors="0">
  <testcase name="LoginTestcase_01" time="18.4" classname="testcases.SignInWithExcelData"/>
  <testcase name="LoginTestcase_02" time="19.27" classname="testcases.SignInWithExcelData">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at testcases.SignInWithExcelData.LoginTestcase_02(SignInWithExcelData.java:90)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- LoginTestcase_02 -->
</testsuite> <!-- Sign in Testcases -->
